packages/indexer/src/handlers/events/evmHost/postRequestHandled.event.handler.ts

```diff
@@ -38,6 +41,25 @@ export const handlePostRequestHandledEvent = wrap(async (event: PostRequestHandl
 			status: Status.DESTINATION,
 			transactionHash,
 		})
+
+		if (transaction &&  transaction.logs) {
+			for (const log of transaction.logs) {
```

Comment:
This function will. be called for each post request handled event in the transaction, there could be multiple, you have to note transactions that have been processed so you skip them in subsequent calls to this function.